deploy/olm-catalog/akka-cluster-operator/1.0.0/akka-cluster-operator-certified.v1.0.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,327 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: akka-cluster-operator-certified.v1.0.0
+  namespace: placeholder
+  annotations:
+    capabilities: Seamless Upgrades
+    categories: Application Runtime
+    certified: 'true'
+    containerImage: 'registry.connect.redhat.com/lightbend/akka-cluster-operator-certified:0.2.2'
+    createdAt: '2019-06-28T15:23:00Z'
+    description: Run Akka Cluster applications on Kubernetes.
+    repository: https://github.com/lightbend/akka-cluster-operator
+    support: 'Lightbend, Inc.'
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "app.lightbend.com/v1alpha1",
+          "kind": "AkkaCluster",
+          "metadata": {
+            "name": "akka-cluster-demo"
+          },
+          "spec": {
+            "replicas": 3,
+            "template": {
+              "spec": {
+                "containers": [
+                  {
+                    "name": "main",
+                    "image": "registry.connect.redhat.com/lightbend/akka-cluster-demo:1.1.0",
+                    "readinessProbe": {
+                      "httpGet": {
+                        "path": "/ready",
+                        "port": "management"
+                      },
+                      "periodSeconds": 10,
+                      "failureThreshold": 10,
+                      "initialDelaySeconds": 20
+                    },
+                    "livenessProbe": {
+                      "httpGet": {
+                        "path": "/alive",
+                        "port": "management"
+                      },
+                      "periodSeconds": 10,
+                      "failureThreshold": 10,
+                      "initialDelaySeconds": 20
+                    },
+                    "ports": [
+                      {
+                        "name": "http",
+                        "containerPort": 8080
+                      },
+                      {
+                        "name": "remoting",
+                        "containerPort": 2552
+                      },
+                      {
+                        "name": "management",
+                        "containerPort": 8558
+                      }
+                    ]
+                  }
+                ]
+              }
+            }
+          }
+        }
+      ]
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+      - kind: AkkaCluster
+        name: akkaclusters.app.lightbend.com
+        version: v1alpha1
```

Comment:
Any plans to bump this up to beta at least now?

Reply:
@edwardcallahan when you said removing alpha labels were you meaning changing this? Or something else?

Reply:
Good question. I specifically left the protocol version at `v1apha1` as to not commit us to more than alpha. Yes, this should move out of alpha. I only changed the release version to 1.0.0 and removed Alpha from the labels on the Marketplace. Again, seeking to restrain commitments to minimal needed for marketplace and avoiding binding us to the protocol definition.